### DISCUSSION TITLE
Does anyone have HDMI/DP audio issue with AppleALC 1.6.2 and above like me?

### DISCUSSION THREAD

**Discussion by goodbest** [osy86/HaC-Mini#619] [Q&A]
My nuc8i7hvk connects with an LG 4k monitor via DP/HDMI.
And the monitor connects to a sound speaker via 3.5mm jack port on the monitor.

With AppleALC 1.6.1, it's working with no issue.
With AppleALC 1.6.2 (commit `2d777ea` to be exactly) or higher , my nuc cannot detect the monitor as a sound speaker.
(DP and Type-C and HDMI are all tested).

I think that this commit saying `Fixed HDMI Audio 0xA171  for Acer VX5-591G` destroys my nuc's hdmi audio.
https://github.com/acidanthera/AppleALC/commit/2d777eab50424b4ba2829dd42da94ecaf98f0299

Does anyone else having this issue?
If so, I will report the issue to acidanthera

(up = AppleALC 1.6.2, bottom=AppleALC=1.6.1)
<img width="616" alt="Screen Shot 2021-08-15 at 19 58 06" src="https://user-images.githubusercontent.com/1118401/129477908-681eb270-3e29-464a-bb44-eb064ddb1216.png">
<img width="627" alt="Screen Shot 2021-08-15 at 20 04 09" src="https://user-images.githubusercontent.com/1118401/129477910-1eab022d-499e-40be-b380-281c85f06236.png">

Background knowledge: Nuc8i7HVK uses 8086-A171 as HDMI audio controller.
<img width="485" alt="Screen Shot 2021-08-15 at 20 06 57" src="https://user-images.githubusercontent.com/1118401/129477976-704bafdf-b9fe-448c-a067-0e2a7607abd2.png">

**Comment by osy** [comment]
@goodbest I haven’t tested but you should report it to acidanthera. That change looks sus to me. Btw this is why I’ve stopped blindly updating the drivers every time because stuff like this happens.
